### ISSUE TITLE
Create password for new internal user using less steps

### ISSUE THREAD

**Issue by nmadruga** [openmsupply/conforma-web-app#1466]
Explained in our private repo: https://github.com/openmsupply/conforma-templates/issues/49

Basically instead of the current existing steps of:
1. As admin submit new internal user details
2. As new user open the email - click on the link 
3. On the Password reset page write your username & submit
4. Open the email again and click on the new link
5. On the Password reset page finaly add your password & submit

It should be less steps:
1. As admin submit new internal user details
2. As new user open the email - click on the link
3. On the password reset page enter the new password & submit

**Comment by CarlosNZ** [comment]
This should be just config, right? I wonder why we haven't done it already.

**Comment by nmadruga** [comment]
> This should be just config, right? I wonder why we haven't done it already.

No.... unfortunately. It would be great to have this simplified, but as I tagged, I think it's work for the next 12 months.
I think the problem is more complex than it seems, because we use the user's session to verify.

**Comment by CarlosNZ** [comment]
> I think the problem is more complex than it seems, because we use the user's session to verify.

Hmm, that shouldn't be required for a verification to work. I might have a quick look at this and see if there's a hurdle I've forgotten about.
